examples/go-libp2p-server/go.mod

```diff
@@ -0,0 +1,111 @@
+module github.com/little-bear-labs/js-libp2p/go-libp2p-server
+
+go 1.18
+
+replace github.com/libp2p/go-libp2p => github.com/ckousik/go-libp2p v0.23.3-0.20221029141116-67b7c6290b11
```

Comment:
We will need to update this once the Go PR is merged.  I'll add an issue for it.

Reply:
Link to the issue in a comment perhaps with a big fat TODO?